app/controllers/nsm/claims_controller.rb

```diff
@@ -32,6 +32,16 @@ def draft
       render 'index'
     end
 
+    def confirm_delete
+      @model = Claim.for(current_provider).find(params[:id])
+    end
+
+    def destroy
+      @model = Claim.for(current_provider).find(params[:id])
```

Comment:
I think a little system spec walking through the user journey for deleting a claim would be ideal.

Reply:
Yeah I realised I missed tests, just writing some up now 😄